dev-resources/less/pages/tyomaapaivakirja.less

```diff
@@ -0,0 +1,107 @@
+.paivakirja-toimitus {
```

Comment:
Yleisesti ottaen välistyksissä on tyylinä käyttää pikseleiden sijasta @valistys-[2,4,8,16,32,jne..]-muuttujia. Nämä ovat remiksi muutettuja pikselimääriä, jotka kestää hieman paremmin skaalausta. Lisäksi tyyleissä pitäisi käyttää ensisijaisesti nimenomaa näitä pikselimääriä, eikä esim. 10px. Poikkeuksia toki on, eikä niille voi mitään. Tarkistaisin designerilta kuitenkin, pitäisikö joidenkin 10px välistysten olla mieluummin 8 tai 12.

Reply:
Joo. Korjattu suurimmaksi osaksi. En edes tuommoisesta tiennyt.

Designerin kommentti
`**8px baseline grid ja siellä saa jossain pienemmissä käyttää 4px**`

https://github.com/finnishtransportagency/harja/pull/2889/commits/0673e2d22d872a6646cc5448012c667797bf4b6b